### PR TITLE
Add Synapse to CDS page

### PR DESCRIPTION
**Pull request by adamjtaylor**
Adds embedded video walkthrough and step-by-step guide for bringing open access data from the HTAN Data Portal to SB-CGC using the synapse python client

Draft of material here: https://docs.google.com/document/d/1UOciskyJCrKj2ABsSuMSL7gb_vp-EXfvzrf3i95EllY/edit?usp=sharing

### CODE REVIEW

File: open_access/synapse_to_cds.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+
+---
+order: 993
```
Comment (adamjtaylor) [inline open_access/synapse_to_cds.md:3]: Unsure how this needs to be set

File: open_access/synapse_to_cds.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+
```
Comment (adamjtaylor) [inline open_access/synapse_to_cds.md:1]: Woops empty line

File: open_access/synapse_to_cds.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+---
+order: 993
+---
+
+# Guide: Transferring Uncontrolled Data from the HTAN Data Portal to the Cancer Genomics Cloud (CGC)
+
+## Screen Recording Walkthrough
+**[HTAN Data Jamboree Tutorial: Getting Uncontrolled Data from the HTAN Portal to SB-CGC](#)**
```
Comment (adamjtaylor) [inline open_access/synapse_to_cds.md:9]: missing a link in case the embed does not work

Comment (adamjtaylor) [reply open_access/synapse_to_cds.md:9]: https://us02web.zoom.us/clips/share/JNSyNG3rPRoWVYjWrYNiV-Z9Tv-TYIKdb54YLB6pk6CRhhnEQ0g14a4-2hItZXGOYXq3CzcSVjgNTL7HMhHjUm72.cd78QzhYRY4j02Mw

**Review by jen-dfci — CHANGES_REQUESTED** [review]
This is great!  Thank you for making the video.

- would change this to order: 994 and make the big_query page 993 so that this page appears right under accessing images on SBG-CDC.  
- would refer to the data as 'open access' rather than 'uncontrolled' to be consistent with the rest of the manual.
- Would shorten "# Guide: Transferring Uncontrolled Data from the HTAN Data Portal to the Cancer Genomics Cloud (CGC)" since this is what appears in the side bar and may be too long.  Perhaps just "Transferring Open Access Data to the Cancer Genomics Cloud (CGC)".  Could we then change the Accessing Images page to "Accessing Images on the Cancer Genomics Cloud (CGC)"?  We need some way to easily distinguish between these two pages.
- On the same note, under line 6, it would help to have a couple of sentences explaining what this is for/how it differs from Accessing Images. Ideally, it would state the type of open access data relevant for this page and distinguish this from images (and provide link to that page) and controlled access (and provide link to that information).

**Comment by adamjtaylor** [comment]
Thanks @jen-dfci those changes have been made

**Review by jen-dfci — APPROVED** [review]

**Pull request merged**